src/azul/service/manifest_service.py

```diff
@@ -765,9 +764,8 @@ def convert(field_name, field_value):
             else:
                 return str(field_value)
 
-        def validate(field_value: str) -> str:
-            assert stripped_joiner not in field_value
-            return field_value
+        # FIXME: Disallow || joiners in metadata; revert PR for (#3888)
+        # 		 https://github.com/DataBiosphere/azul/issues/3911
```

Comment:
The link that's in the comment is not align indented, there is an extra space before the `https`

Reply:
Consider leaving the `validate` method in place, but adding a `if False` at the start of the body, that would make it more aligned with the [contributing guidelines](https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#118disabling-sections-of-code). 

By removing the method the FIXME looks a bit out of place.